thread.go

```diff
@@ -88,11 +88,21 @@ const (
 )
 
 type ThreadMessage struct {
-	Role        ThreadMessageRole  `json:"role"`
-	Content     string             `json:"content"`
-	FileIDs     []string           `json:"file_ids,omitempty"`
-	Attachments []ThreadAttachment `json:"attachments,omitempty"`
-	Metadata    map[string]any     `json:"metadata,omitempty"`
+	Role        ThreadMessageRole      `json:"role"`
+	Content     []ThreadMessageContent `json:"content"`
+	Attachments []ThreadAttachment     `json:"attachments,omitempty"`
+	Metadata    map[string]any         `json:"metadata,omitempty"`
+}
+
+type ThreadMessageContent struct {
+	Type      string                  `json:"type"`
+	ImageFile *ThreadMessageImageFile `json:"image_file,omitempty"`
+	Text      *string                 `json:"text,omitempty"`
```

Comment:
Should `omitempty` potentially be enough here without pointer usage?

Reply:
Yea, fixed.